var/spack/repos/builtin/packages/hpe-mpi/package.py

```diff
@@ -32,7 +32,7 @@ class HpeMpi(Package):
             sha256='e067e4ba382d306d540e1ad5bcd63035a4aa93e7d7c15d617e82de4160e5ad8a',
             url='file:///gpfs/bbp.cscs.ch/apps/hpc/download/hpe-mpi/hpe-mpi-2.22.hmpt.tar.xz')
     version('2.25.hmpt',
-            sha256='70ceccca4dccd1865e778b01a631c7e85ea0913d7dbea6608d59ad4be3738187',
+            sha256='126a46bb2cbd4b63bd7b3aed74cee5e8d08e166e9748071fd0b308be29335e1a',
```

Comment:
Sha change but no version change?

Reply:
@matz-e This is a patch to 2.25. It is technically still 2.25 but ideally with IME fixed. I'm going to test now :)

Reply:
IIRC, this is not going to rebuild anything, though. For that, the version has to change.

Reply:
Will this happen even if we run the incubator job trick?

Reply:
Sure, you can do that…

Reply:
Done, let me ask Jenkins to redeploy ;)

Reply:
> IIRC, this is not going to rebuild anything, though. For that, the version has to change.

Note that:
* in actual deployment we are changing from 2.22 to 2.25
* only my last commit where I changed SHA without version but before doing that I did necessary incubator job trick.